swim/member.go

```diff
@@ -107,6 +110,8 @@ func statePrecedence(s string) int {
 		return 2
 	case Leave:
 		return 3
+	case Tombstone:
```

Comment:
I don't think there'll be much motivation to refactor it if we don't do it now.

If you think it's too big though, then I'd say yeah, let's leave it.